### ISSUE TITLE
Alt text for conceptual models

### ISSUE THREAD

**Issue by kimberly-bastille** [NOAA-EDAB/tech-doc#70]
Tech doc's 508 compliance check is failing on alt text for the conceptual models. I am not sure how to write this alt text and there are quite a few visualizations in the [conceptual model chapter](https://noaa-edab.github.io/tech-doc/conceptual-models.html). Any advice? @sgaichas

**Comment by kimberly-bastille** [comment]
Conceptual model alt text can be found in the [alt text spreadsheet](https://docs.google.com/spreadsheets/d/1CVShvuH-SDwWoZdNc_h9dx1Q3EwCk1YFpiXLs9saLys/edit#gid=0). Example below:

Box and wire diagram of linkages between human activities, marine habitat, the environment, and the marine ecological community.  Human activities include commercial and recreational fisheries and tourism, and human social-cultural practices and attachments are included. Habitat includes pelagic, seafloor/demersal, nearshore, and freshwater/estuary. Environmental factors are generalized in a climate factor. The focal species in the marine communities are protected species, forage fish, groundfish, and fished invertebrates. Supporting species in the marine environment include zooplankton, primary production, and benthos.
